static/app/views/alerts/rules/metric/wizardField.tsx

```diff
@@ -176,6 +179,19 @@ export default function WizardField({
           : []),
       ],
     },
+    ...((hasLogAlerts(organization)
+      ? [
+          {
+            label: t('LOGS'),
+            options: [
+              {
+                label: AlertWizardAlertNames.trace_item_logs,
+                value: 'trace_item_logs' as const,
+              },
+            ],
+          },
+        ]
+      : []) as GroupedMenuOption[]),
```

Comment:
I think this value already satisfies the type and we can probably drop the type assertion here